README.md

```diff
@@ -253,13 +275,22 @@ issue will not discover the information in there.
 
 If reporting a problem as a new issue, please use the appropriate
 [bluez-alsa GitHub issue reporting template][] and complete each section of
-the template as fully as possible.
-
+the template as fully as possible. Note that the `Bug report` template should
+be used for all questions relating to a specific installation of BlueALSA, even
+if you are not sure that it is a bug (the name of the template is set by
```

Comment:
I don't think that it's set by GitHub... It's here: https://raw.githubusercontent.com/arkq/bluez-alsa/refs/heads/master/.github/ISSUE_TEMPLATE/1-bug-report.md and we can change it (and the name of the file itself). If you feel that `Problem report` is better wording (I'm not a native speaker, so for me bug/problem/issue is all the same more or less :D) you can update the template.

Reply:
I've had a look around some other projects and it seems that "Bug report" is very common, so perhaps that is best left as it is. I've removed the comment about github.

I'm thinking maybe the blank template should be removed, since so many issues are created with this when really the bug report template would have been more useful?

Reply:
I'm not sure whether the blank one can be removed. I don't think that it's configured anywhere...

Reply:
After some more thought, perhaps the blank template should remain, at least until there has been time to assess whether the new README text is helping. If it is ever decided to remove it, then I think there must be a way, because there are some other projects that have done so.

Reply:
Indeed it's possible to disable that blank template: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser